test/model/test_store.dart

```diff
@@ -22,7 +23,15 @@ import '../example_data.dart' as eg;
 ///
 /// See also [TestZulipBinding.globalStore], which provides one of these.
 class TestGlobalStore extends GlobalStore {
-  TestGlobalStore({required super.accounts});
+  TestGlobalStore({required super.globalSettings, required super.accounts})
+    : _globalSettings = globalSettings;
+
+  GlobalSettingsData? _globalSettings;
```

Comment:
```suggestion
  GlobalSettingsData _globalSettings;
```

right?

Reply:
Hmm or: what if we just leave this field out? Seems like the base class takes care of it.

And in the case of accounts we don't find we need a corresponding field here.

Reply:
Yeah. It should be very similar to `doUpdateAccount`:

```dart
  @override
  Future<void> doUpdateAccount(int accountId, AccountsCompanion data) async {
    // Nothing to do.
  }
```